docs/setup.md

```diff
@@ -9,7 +9,25 @@ This document outlines the setup process for the `Superposition Platform`.
 - Install wasm-pack
     `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
 - Install docker
-- make sure you have dependencies like libpq, openssl, libiconv,etc installed
+- make sure you have dependencies like libpq, openssl, libiconv, diesel_clietc installed
+- To install these dependencies run the following commands -
+    ```bash
+    brew install libpq # Add libpq to the path
```

Comment:
Can we make a single brew command to install all deps to reduce the number of steps?

Reply:
Done